website/blog/2022-01-11-aluminium.md

```diff
@@ -0,0 +1,734 @@
+---
+author: Tomasz Godzik
+title: Metals v0.11.0 - Aluminium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.11.0! Due to the number of
+changes in this release we decided to introduce one more minor version before
+Metals 1.0.0, which is planned to come this year.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">336</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">136</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">11</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">11</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/45?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Add CFR class file viewer.
+- [Scala 3] Type annotations on code selection.
+- Simplify usage of sbt server.
+- Better support for mill BSP.
+- Extract value code action.
+- Add command to run current file.
+- Support test explorer API.
+- Add go to definition via links for synthetic decorations.
+- Basic Java support.
+- Add support for Scala 2.13.8
+- Reworked Metals Doctor
+
+## Add CFR class file viewer
+
+In addition to the recently added semanticdb, tasty and javap functionalities,
+thanks to the efforts of [Arthurm1](https://github.com/Arthurm1), it's now
+possible to decompile class files using Lee Benfield's
+[CFR (Class File Reader)](https://www.benf.org/other/cfr/faq.html).
+
+CFR decompiles class files to java, which is much more readable than using
+`javap`.
+
+![cfr](https://i.imgur.com/exDFGyT.gif)
+
+In the future this might help in go to definition for jars that do not have a
+corresponding source jar, which is what Metals uses today.
+
+Editors support: Any editor that exposes the command.
```

Comment:
```suggestion
Editor support: Any editor that exposes the command.
```

Reply:
Checked and seems you are right, though emacs is not far behind.

Reply:
Fxied!